packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -47,7 +46,7 @@ const PatientChart: React.FC = () => {
           >
             <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
             <aside>
-              <PatientBanner {...state} />
+              <ExtensionSlot extensionSlotName="patient-header-slot" state={state} />
```

Comment:
Deleted some duplicate code for the patient banner within the patient chart esm and replaced it with an extension to load the version from the patient banner esm

Reply:
Hi @pirupius !
Actually this was done intentionally because as an extension, the patient banner was taking a lot of time to load and we wanted it to be the first thing to be loaded on the patient chart page.

Reply:
@vasharma05 thanks for your feedback. I have however tested this with a backend server with over 350k patients and i haven't yet seen an issue. Could it be that this isn't a blocker anymore?